plugins/xterm/xterm.plugin.bash

```diff
@@ -12,23 +12,23 @@ _omb_module_require plugin:bash-preexec
 
 function _omb_plugin_xterm_short_dirname {
   local dir_name=${PWD/~/\~}
-  if [[ ${OMB_PLUGIN_XTERM_SHORT_TERM_LINE-} == true ]] && ((${#dir_name} > 8)); then
+  if [[ ${OMB_PLUGIN_XTERM_SHORT_TERM_LINE:-} == true ]] && ((${#dir_name} > 8)); then
```

Comment:
Is this change needed?

Reply:
I assumed this is make sure it works with -u option where undefined var rerence is an error. ${var:-} will create valid reference with empty string when 'var' is not defined.

Reply:
> not I cold find any good doc on `${var-}`

It is not listed in the list of parameter expansions in the Bash manual but is described in the text body.

> [1] [3.5.3 Shell Parameter Expansion - Bash Reference Manual](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion) paragraph 6 (**boldface** by me):
>
> When not performing substring expansion, using the form described below (e.g., ‘:-’), Bash tests for a parameter that is unset or null. **Omitting the colon results in a test only for a parameter that is unset.** Put another way, if the colon is included, the operator tests for both *parameter*’s existence and that its value is not null; if the colon is omitted, the operator tests only for existence.

In this case, actually there is no difference in the behavior between `${var-}` and `${var:-}`. However, as far as the intent is to work around `set -u`, `${var-}` is more explicit that it checks whether the variable is unset.

Reply:
Awesome. So I'd revert this too.

Reply:
Thanks. Just in case, the colon is needed for the case of `${1:-xterm}`. You can revert the colons only for two cases of `OMB_PLUGIN_XTERM_SHORT_TERM_LINE`.

Reply:
Thanks!